pxtblocks/fields/field_note.ts

```diff
@@ -50,19 +50,53 @@ namespace pxtblockly {
         GSharp5 = 831,
         A5 = 880,
         Bb5 = 932,
-        B5 = 988
+        B5 = 988,
+        C6 = 1047,
+        CSharp6 = 1109,
+        D6 = 1175,
+        Eb6 = 1245,
+        E6 = 1319,
+        F6 = 1397,
+        FSharp6 = 1480,
+        G6 = 1568,
+        GSharp6 = 1568,
+        A6 = 1760,
+        Bb6 = 1865,
+        B6 = 1976,
+        C7 = 2093
     }
 
-    enum PianoSize {
-        small = 12,
-        medium = 36,
-        large = 60
+    interface NoteData {
+        name: string,
+        prefix: string,
+        altPrefix?: string,
+        freq: number
+    }
+
+    const Notes: {[key: number]: NoteData} = {
+        //TODO fill out the rest of these
+        40: {name: "C", prefix: "Middle", freq: 262},
```

Comment:
Needs to be localizable

Reply:
How was it working before? Doesn't the big comment starting on line 237 take care of that?

Reply:
Could we move to a format similar to: `C (middle)`, `D (Tenor)`, etc? Advantages:

- Enables independent localization of prefix and notes, because it should work in most languages
- We can now programmatically generate this Notes dictionary during initialization instead of manually maintaining it manually
- We no longer need the huge localization comment below (which also needs to be maintained manually)